arionBackend/serializers/json.py

```diff
@@ -4,41 +4,61 @@
 from json import loads as json_loads
 
 
-def serialize_hierarchy_overview(hierarchy):
-	"""
-	Serializes a hierarchy to JSON format with basic information.
-	:param hierarchy: given hierarchy to serialize.
-	:return: Dictionary with hierarchy information
-	"""
-	return {
-		"id": hierarchy.id,
-		"name": hierarchy.name,
-		"timestamp": hierarchy.timestamp
-	}
+class JSONSerializer(object):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a docstring to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1720)